views/database/edit.erb

```diff
@@ -26,7 +25,7 @@
           </td>
         </tr>
         <tr>
-          <td>Project Id</td>
+          <td>Project Name</td>
           <td>
             <% if @db %>
```

Comment:
If a DB exists without a parent project
this line is where the view is trying to parse a nil object

should change to something like `<% if @project %>`